config/environments/development.rb

```diff
@@ -27,7 +27,14 @@
   end
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
+
+  config.action_mailer.delivery_method = :test
+
+  host = "localhost:3000" # Don't use this literally; use your local dev host instead
+  # Use this on the cloud IDE.
+  config.action_mailer.default_url_options = { host: host, protocol: 'http' }
+
```

Comment:
Config gửi mail bằng gmail cho môi trường development nha em
https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration-for-gmail